test/topology/conftest.py

```diff
@@ -190,7 +190,7 @@ async def manager(request, manager_internal, record_property, mode):
                         mode /
                         f"{pathlib.Path(xml_path.stem).stem}.log"
                         )
-    test_log = suite_testpy_log.parent / f"{suite_testpy_log.stem}.{test_case_name}.log"
+    test_log = suite_testpy_log.parent / f"{suite_testpy_log.stem}.{test_case_name}-arst.log"
```

Comment:
What does `-arst` mean and why is it here?

Reply:
Good catch. Actually, this is a leftover from my investigation. Reverted this change.